multi-node/generic/controller-install.sh

```diff
@@ -382,37 +383,87 @@ spec:
 EOF
     fi
 
-    local TEMPLATE=/srv/kubernetes/manifests/kube-dns-rc.yaml
-    if [ ! -f $TEMPLATE ]; then
+    local TEMPLATE=/etc/kubernetes/manifests/calico-policy-controller.yaml
+    if [ "${USE_CALICO}" = "true" ] && [ ! -f "${TEMPLATE}" ]; then
```

Comment:
This change and the change below don't seem right. This might be left over from a rebase (calico stuff recently changed)